app/mixins/github-url-properties.js

```diff
@@ -3,11 +3,13 @@ import { inject as service } from '@ember/service';
 import { computed } from '@ember/object';
 
 export default Mixin.create({
-  externalLinks: service(),
+  vcsLinks: service(),
 
-  urlGithubPullRequest: computed('repo.slug', 'build.pullRequestNumber', function () {
-    let slug = this.get('repo.slug');
-    let pullRequestNumber = this.get('build.pullRequestNumber');
-    return this.get('externalLinks').githubPullRequest(slug, pullRequestNumber);
+  urlGithubPullRequest: computed('repo.{vcsType,slug}', 'build.pullRequestNumber', function () {
```

Comment:
@artursmirnov I couldn't find any instances of usage for this `urlGithubPullRequest` function, could you please confirm whether this is indeed not used?

Reply:
Hey @rafael-travis, this mixin itself is used in [this controller](https://github.com/travis-ci/travis-web/blob/as-update-deps/app/controllers/build.js#L4), but I also did find any occurrences of this property. I'd suggest to try removing the property (and mixin) and run tests. If tests pass, then it's safe to remove it.